assets/containerisableComponents/contributionsThankYou/contributionsThankYouPage.jsx

```diff
@@ -12,16 +11,18 @@ import CirclesIntroduction from 'components/circlesIntroduction/circlesIntroduct
 import QuestionsContact from 'components/questionsContact/questionsContact';
 import SpreadTheWord from 'components/spreadTheWord/spreadTheWord';
 import DirectDebitGuarantee from 'components/directDebit/directDebitForm/directDebitGuarantee';
-import { openDirectDebitGuarantee, closeDirectDebitGuarantee } from 'components/directDebit/directDebitActions';
+import { type Contrib } from 'helpers/contributions';
```

Comment:
`{ type Contrib }` <- did not know you could do this! That'll be handy for simplifying a few imports around the codebase 👍